gemd/entity/setters.py

```diff
@@ -6,7 +6,7 @@
 T = TypeVar('T')
 
 
-def validate_list(obj: Union[Iterable[T], T],
+def validate_list(obj: Optional[Union[Iterable[T], T]],
```

Comment:
Collateral discovery.  It accepts a None, initializing to empty list, so why is that not documented?